ui/app/styles/core/tag.scss

```diff
@@ -52,4 +52,14 @@
   &.is-outlined {
     box-shadow: 0 0 0 1px $white;
   }
+
+  &.is-structure {
+    color: rgb(98, 104, 115);
+    text-transform: none;
+    font-size: 12px;
+
+    width: 48px;
+    height: 16px;
+    border-radius: 0;
```

Comment:
Looking at this in Freestyle, it seems very similar to the existing `badge` style that we get from Bulma. Unfortunately there is no Badge or Tag specific page in Freestyle, but if you look at Timeline, you can see both side-by-side. 

I wonder if we can get away with just using Badge?

Reply:
I’ve switched it to `badge`, thanks.